packages/fluentui/e2e/e2eApi.ts

```diff
@@ -35,47 +35,78 @@ export class E2EApi {
     return await this.page.hover(selector);
   };
 
-  public getAttributeValue = async (selector: string, attr) => {
-    return this.page.$eval(selector, (el, attribute) => el.getAttribute(attribute), attr);
+  public hasComputedStyle = async (
+    selector: string,
+    property: keyof CSSStyleDeclaration,
+    value: string,
+  ): Promise<void> => {
+    await this.page.waitForFunction(
+      (selector, property, value) => {
+        return window.getComputedStyle(document.querySelector(selector))[property] === value;
+      },
+      { timeout: 10 * 1000 },
```

Comment:
you hard code the same timeouts in each of your selectors. Would it make sense to store this value as a const or provide it as some part of config ?

Reply:
extracted to `PUPPETEER_ACTION_TIMEOUT`